web/src/reducers/dataReducerForHistoryFeature.js

```diff
@@ -19,7 +19,6 @@ Object.entries(zonesConfig).forEach((d) => {
   zone.capacity = zoneConfig.capacity;
   zone.contributors = zoneConfig.contributors;
   zone.timezone = zoneConfig.timezone;
-  zone.shortname = translation.getZoneNameWithCountry(key);
```

Comment:
As far as I can tell this was only used in `zonelist.jsx` and was always identical to the countryKey so I removed it in favor of using the countryKey.